cpp/include/cudf/join.hpp

```diff
@@ -500,6 +500,25 @@ class distinct_hash_join {
   inner_join(rmm::cuda_stream_view stream        = cudf::get_default_stream(),
              rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource()) const;
 
+  /**
+   * Returns the build table indices that can be used to construct the result of performing
+   * a left join between two tables
```

Comment:
Please add `@brief` and a period in the `inner_join` docs, too.
```suggestion
   * @brief Returns the build table indices that can be used to construct the result of performing
   * a left join between two tables.
```

Reply:
349d853